package.json

```diff
@@ -26,16 +26,26 @@
   "homepage": "https://github.com/UKHomeOffice/lev-react-components#readme",
   "devDependencies": {
     "@babel/core": "^7.4.5",
-    "@babel/preset-env": "^7.4.5",
-    "@babel/preset-react": "^7.0.0",
+    "@babel/preset-env": "^7.8.7",
+    "@babel/preset-react": "^7.8.3",
     "@svgr/rollup": "^4.3.0",
+    "@testing-library/jest-dom": "^5.1.1",
+    "@testing-library/react": "^9.5.0",
+    "babel-jest": "^25.1.0",
     "babel-plugin-styled-components": "^1.10.0",
+    "babel-polyfill": "^6.26.0",
+    "babel-preset-react": "^6.24.1",
+    "enzyme": "^3.11.0",
     "govuk-react": "^0.7.1",
+    "jest": "^25.1.0",
     "moment": "^2.24.0",
-    "nodemon": "^1.19.1",
+    "node-run-cmd": "^1.0.1",
```

Comment:
Is this being used?

Reply:
No good spot!

Reply:
Fixed

Reply:
There are quite a few new packages added in this commit. It looks like you were experimenting with different things. I think it's probably worth looking through those changes and making sure that everything in there is actually required.

Reply:
I've reviewed the packages, there should just be the ones we need there now. Fixed